Create Rule “technical-overview/rule” (#2962)
SSWConsulting/SSW.Rules.Content: rules/technical-overview/rule.md

---
type: rule

> title: Rules to having a good technical overview

Reviewer: Rule titles are written in this format
`Do you ... ?`

```suggestion
title: Do you have a good technical overview?
```

Its also good to add a prefix 
e.g. Twitter - Do you know to thank others for each reference to you?
https://www.ssw.com.au/rules/thank-others-for-each-reference-to-you

uri: technical-overview
authors:
  - title: Bryden Oliver
    url: https://www.ssw.com.au/people/bryden-oliver
  - title: Ulysses Maclaren
    url: https://www.ssw.com.au/people/uly
  - title: Patrick Zhao
    url: https://www.ssw.com.au/people/patrick-zhao
  - title: Adam Cogan
    url: https://www.ssw.com.au/people/adam-cogan
created: 2022-04-21T23:39:49.013Z
guid: e9a156f4-f518-4ea9-924a-42811ee10e48
---

> When running a project, having someone with a wide overview is extremely valuable. That person is much more likely to notice broader problems and be able to act to correct them. 

Reviewer 2: What is a wide overview? Maybe better define that

It is highly effective to have this person attend sprint reviews and planning. Attending daily scrums is also valuable but not essential, even attending occasionally has value. 

This person's objectives should include the following:
1. Provide feedback to the Product Owner on what’s in backlog

>     - Are items sensibly sized?

Reviewer 2: Make clear that "sensibly sized" is about breaking down tasks, not critiquing developer estimations

>     - Are their risky items in the backlog that could do with a SPIKE to research the risks?

Reviewer 2: What is a SPIKE? Maybe better define that

    - Do the items address the goals of the project? (you'd be surprised how often the answer is no to this one)
2. Provide feedback to Product Owner on prioritization
    - Often a Product Owner will feel everything is most important get them to list things in order
    - Review prioritization just before sprint review
    - Try and provide tips on how to think about prioritization (asking questions like "Is a more important than b", "If yes, we should move a above b in the backlog")

3. Provide feedback to Scrum Master on how things are running
    - Often the Scrum Master will miss things due to being bogged down
    - Remind Scrum Master their role is to facilitate and assist in eliminating blockers
4. Give big picture feedback highlighting the requirements that often get missed, encouraging the Product Owner to decide which items are most important in their context. The most common of these are:
    - Performance
    - Cost

> 

Reviewer 2: why a random line break

>     - Cost
> 
>     - Scale

Reviewer: Remove empty line

```suggestion
    - Cost
    - Scale
```
